azkustoingest/streaming.go

```diff
@@ -171,6 +189,10 @@ func (i *Streaming) newProp() properties.All {
 	}
 }
 
+func (i *Streaming) QueryClient() QueryClient {
```

Comment:
Why is this a public API?
Do we need it for something other than tests?

Reply:
Getting access to the client is useful, otherwise to run commands like changing the batching will require creating a new one.
It does save work on tests, but I think it's useful regardless.